engine/runtime/src/bench/scala/org/enso/interpreter/bench/IRUtils.scala

```diff
@@ -1,8 +1,7 @@
 package org.enso.interpreter.bench
 
-import org.enso.compiler.core.IR
-
 object IRUtils {
```

Comment:
There `IRUtils` can now be removed from this PR, I assume.

Reply:
Sure, done in 0ee38dfb8444eae3e841a578916b6148f1d611e9